default_package/Gemfile

```diff
@@ -1,8 +1,8 @@
 source "https://rubygems.org"
 
 # Mandatory minimal gems for Scarpe-Wasm and for packaging
-gem "scarpe-components", github: "scarpe-team/scarpe", glob: "scarpe-components/*.gemspec"
-gem "lacci", github: "scarpe-team/scarpe", glob: "lacci/*.gemspec"
+gem "scarpe-components", github: "scarpe-team/scarpe",branch:"main", glob: "scarpe-components/*.gemspec"
```

Comment:
just some thought came to mind but i think it would be nice to have branches as releases? like branch: 2.0-stable or something ?

Reply:
We'll eventually want release branches, yeah. Right now, scarpe-wasm is really not mature enough for that yet...